sansa-query-spark/src/main/scala/net/sansa_stack/query/spark/dof/bindings/Bindings.scala

```diff
@@ -0,0 +1,49 @@
+package net.sansa_stack.query.spark.dof.bindings
+
+import net.sansa_stack.query.spark.dof.node._
+import net.sansa_stack.query.spark.dof.tensor.Tensor
+import org.apache.jena.graph.Triple
+import scala.reflect.ClassTag
+
+/*
+ * Class used to bind variables with their values
+ */
+class Bindings[R, N: ClassTag, T, A](model: Tensor[R, N, T, A], constraints: Constraints) extends Serializable {
+  private var _mapV = new VariableMap[N](model.sparkContext, constraints)
+  def mapV: VariableMap[N] = _mapV
+
+  private var _result: Result[A] = _
+  def result: Result[A] = _result
+  def result_=(value: Result[A]): Unit = _result = value
+
+  def getModel: Tensor[R, N, T, A] = model
+
+  def saveResult(triple: Triple, rdd: A): Unit = {
+    val start = Helper.start
+    result = model.saveResult(triple, result, rdd)
+    Helper.measureTime(start, s"\nSave result time=")
+  }
+
+  def recalcDof(triples: DofTripleList): List[(Int, Triple)] = {
+    def calcDof(triple: Triple) = {
+      // how many constants (or variables to which there exists a non-empty set associated in V)
```

Comment:
Did you think of providing a multiple column comments ? eg. 
```scala
/** 
  * recalcDof method checks ....
**/
```

Reply:
done